<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalyticsSdkEventWidget</class>
 <widget class="QWidget" name="AnalyticsSdkEventWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>92</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
     <item row="1" column="1">
      <widget class="QLineEdit" name="captionEdit">
       <property name="placeholderText">
        <string>Keywords separated by space</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
       <widget class="nx::vms::client::desktop::LabelWithHint" name="captionLabel">
         <property name="text">
           <string>Caption contains:</string>
         </property>
       </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="descriptionEdit">
       <property name="placeholderText">
        <string>Keywords separated by space</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
       <widget class="nx::vms::client::desktop::LabelWithHint" name="descriptionLabel">
         <property name="text">
           <string>Description contains:</string>
         </property>
       </widget>
     </item>
     <item row="0" column="1">
      <widget class="QnTreeComboBox" name="sdkEventTypeComboBox"/>
     </item>
     <item row="0" column="0">
       <widget class="nx::vms::client::desktop::LabelWithHint" name="sdkEventTypeLabel">
         <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
           </sizepolicy>
         </property>
         <property name="text">
           <string>Event Type:</string>
         </property>
       </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
    <class>nx::vms::client::desktop::LabelWithHint</class>
    <extends>QLabel</extends>
    <header location="global">nx/vms/client/desktop/common/widgets/widget_with_hint.h</header>
  </customwidget>
  <customwidget>
   <class>QnTreeComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">ui/widgets/common/tree_combo_box.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
